src/components/cursor.js

```diff
@@ -89,10 +89,10 @@ module.exports.Component = registerComponent('cursor', {
     if (this.data.rayOrigin === 'entity') { this.resetRaycaster(); }
     this.updateMouseEventListeners();
     // Update the WebXR event listeners if needed
-    if (this.data.rayOrigin === 'xrselect') {
+    if (this.data.rayOrigin !== 'mouse') {
```

Comment:
This should probably be explicit. If in the future we add an option to `rayOrigin` this might be no longer correct.

```
var rayOrigin = this.data.rayOrigin;
if (rayOrigin === 'entity' || rayOrigin === 'xrselect') {
  this.addWebXREventListeners();
}
```